js/pages/samples.js

```diff
@@ -314,19 +313,26 @@ const sanford = {
     whatHappens: 'Check in at the front desk when you enter the Imagenetics Building. The registration team will direct you to the lab where the Connect team will collect your samples. At the end of your visit, the Connect team will check you out of your appointment.<br><br>We will draw a blood sample, collect a urine sample, and collect a saliva sample by asking you to swish with mouthwash.<br><br>We will also ask you to complete a short survey on the MyConnect app using your mobile phone. You will need your MyConnect app login information to complete the survey. If you do not have a mobile phone, we may be able to provide you with a tablet to complete your survey.<br><br>We strongly encourage you to complete your survey at your appointment. If you choose to complete it after you leave your appointment, it is important to do so as soon as possible.',
     support: 'Call 1-877-505-0253 (8:00 a.m.-10:00 p.m. CT on weekdays and 9:00 a.m.-6:00 p.m. CT on weekends)',
     locations: [
         [
-            'Sanford Imagenetics',
-            '1321 W. 22nd St.<br>Sioux Falls, SD 57105',
-            'Monday - Thursday: 7:00 a.m.-4:00 p.m.<br>Friday: 7:00 a.m.-2:00 p.m.',
-            'Patient parking available in lot.'
-        ]
+            'Sioux Falls: Sanford Imagenetics',
+            '1321 W. 22nd St.<br>Sioux Falls, SD 57104',
+            '',
+            'Free patient parking available is available in the lot near the front entrance--Door JJ. No parking validation needed.'
+        ],
+
+        [
+           'Fargo: South University Urgent Care/Orthopedic Walk-In Clinic',
+           '1720 University Drive S<br>Fargo, ND 58102<br> Enter Through Door #8',
```

Comment:
I think the space after the `<br>` and before 'Enter Through Door #8' needs to be removed.
`'1720 University Drive S<br>Fargo, ND 58102<br>Enter Through Door #8',`

Reply:
Agreed, but I'm referring to the space directly after the 2nd `<br>`. There is a difference between your code on line 332 and my code comment. Thanks.